src/Samples/SarifToCsv/Program.cs

```diff
@@ -23,13 +23,24 @@ namespace SarifToCsv
     /// </summary>
     public class Program
     {
+        public class WriteContext
```

Comment:
I didn't mean for the SarifToCsv work to be in this PR. I made a branch from the in progress branch, merged another, and then pushed, and it updated the branch from the PR. :/